projects/example-app/src/app/books/containers/selected-book-page.component.ts

```diff
@@ -12,7 +12,7 @@ import * as fromBooks from '@example-app/books/reducers';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <bc-book-detail
-      [book]="book$ | async"
+      [book]="(book$ | async)!"
```

Comment:
may be the `book` should have its type changed to support a falsy value since the book may not be initialized